src/content/tools/developer-tools/json.md

```diff
@@ -29,4 +29,19 @@ The five buttons only work when the JSON is valid. If there are syntax errors, t
 Sometimes you will want to convert JSON data into static strings in your C++ application firmware. The
 tool below will convert the JSON above into code.
 
-{{> json-to-code rows="15" cols="90"}}
+{{> json-to-code rows="10" cols="90"}}
+
+### Convert to CBOR
+
+CBOR is a compact binary representation of structured data that can be converted to and from JSON.
+
+{{> json-to-cbor rows="10" cols="90"}}
```

Comment:
What's the use case for this tool? We don't expose CBOR directly to the user. It's a transport optimization for Variant.

Reply:
It's so users can see how big they can make their data and still have it fit in the current 1024 byte limit.